styles.css

```diff
@@ -0,0 +1,409 @@
+              /  \              (oo)
+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+           Cow swimming at Amity Island
+(Where Jaws was set, for those less educated)*/
+
+/* HERO STYLES */
+
+.hero {
+	margin: 0;
+	color: #ffffff;
+	background-color: #000000;
+	background-image: url(images/hero.jpg);
+	background-position: center;
+  	background-repeat: no-repeat;
+  	background-size: cover;
+  	position: relative;
+	height: 870px;
+}
+
+#hero-text {
+	margin: 0 0 0 20px;
+	font-size: 46px;
+	font-family: 'Lora', serif;
+	position: relative;
+	top: 550px;
+	left: 50px; 
+}
+
+.hero h1 {
+	margin: 0;
+	font-family: 'Lora', serif;
+}
+
+.hero h3 {
+	margin: 0;
+	font-size: 95%;
+}
+
+.hero h6 {
+	margin: 0;
+	font-size: 40%;
+}
+
+#hero-arrow {
+	display: inline-block;
+	position: relative;
+	text-decoration: none;
+	color: #ffffff;
+	font-size: 50px;
+	font-weight: 700;
+	top: 615px;
+	left: 48%; 
+}
+
+#hero-arrow:hover {
+	color: #5f5f5f;
+}
+
+/*          (__)
+            ($$)
+     /-------\/
+    / |=====||
+   *  ||----||
+      ~~    ~~
+This cow is a Yuppie*/
+
+/* ABOUT STYLES */
+
+#about-text {
+	text-align: center;
+	padding: 52px;
+}
+
+#about-text h1 {
+	font-size: 44px;
+	margin: 0;
+	padding-top: 20px;
+}
+
+#about-text p {
+	font-size: 28px;
+	font-weight: 200;
+}
+
+/*               \_|_/
+                  (oo)
+           /-------\/
+          / |     ||
+         *  ||----||
+            ~~    ~~
+Cow visiting the Statue of Liberty
+*/
+
+/* PORTFOLIO ITEMS STYLES */
+
+.portfolio {
+	background-color: #000000;
+	color: #ffffff;
+}
+
+.portfolio .container {
+	padding-top: 30px;
+}
+.portfolio h2 {
+	margin: 0 0 20px 0;
+	text-align: center;
+	font-size: 54px;
+}
+
+.portfolio h3 {
+	font-size: 25px;
+	margin-bottom: 5px;
+}
+
+.portfolio-text {
+	padding-top-top: 20px;
+}
+
+#portfolio-item-box {
+	text-align: center;
+}
+
+.portfolio-item {
+	display: inline-block;
+	margin: 20px auto 5px auto;
+	margin-right: 5px;
+	margin-left: 5px;
+}
+
+.portfolio-item div {
+	width: 365px;
+	height: 225px;
+	background-color: #fefefe;
+	border-radius: 10px;
+}
+
+.portfolio-item a {
+	font-size: 20px;
+}
+
+/*     \^^^^^^^^\   (__)
+        \^^^^^^^^\\ (oo)
+     *-----\_______\/\/
+   ^_______/   ---  \______^
+  ^--------\   \S/  /\_____^
+            \______/
+
+It's a bird...
+ It's a plane...*/
+
+/*PORTFOLIO ITEM IMAGES*/
+
+.portfolio-item div {
+	background-position: center;
+  	background-repeat: no-repeat;
+  	background-size: cover;
+  	position: relative;
+}
+
+.coming-soon {
+	background-image: url(images/coming-soon.png);
+}
+
+#pf-item-1-img {
+	background-image: url(images/item1.png);
+}
+
+
+/*                   (__)
+                     (oo)
+  /-------------------\/
+ / |                 ||
+*  ||----------------||
+   ^^                ^^
+Stretch Cow*/
+
+/* CONTACT STYLES */
+.contact h2 {
+	text-align: center;
+}
+
+.contact article {
+	display: inline-block;
+	text-align: center;
```

Comment:
not doing anything

Reply:
yes it does, aligns text only visible on hover